lib/pacemaker/pcmk_output.c

```diff
@@ -677,7 +677,7 @@ health_xml(pcmk__output_t *out, va_list args)
     pcmk__output_create_xml_node(out, pcmk__s(sys_from, ""),
                                  "node_name", pcmk__s(host_from, ""),
                                  "state", pcmk__s(fsa_state, ""),
-                                 "result", pcmk__s(result, ""),
+                                 PCMK_XA_RESULT, pcmk__s(result, ""),
```

Comment:
I suppose this usage is why it's public? Makes sense ...

Reply:
I'm making public anything that's in an API schema, unless you object.

Reply:
Makes sense to me